sources/packages/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.controller.ts

```diff
@@ -51,21 +52,34 @@ export class ConfirmationOfEnrollmentController {
    * @returns student application list of an institution location
    */
   @HasLocationAccess("locationId")
-  @Get(":locationId/confirmation-of-enrollment")
+  @Get(
+    ":locationId/confirmation-of-enrollment/enrollmentPeriod/:enrollmentPeriod",
+  )
   async getCOESummary(
     @Param("locationId") locationId: number,
+    @Param("enrollmentPeriod") enrollmentPeriod: EnrollmentPeriod,
   ): Promise<COESummaryDTO[]> {
-    const applications = await this.applicationService.getCOEApplications(
-      locationId,
-    );
-    return applications.map((eachApplication: Application) => {
+    if (
+      enrollmentPeriod !== EnrollmentPeriod.Current &&
```

Comment:
Just a alternate way to not check every single item of the enum one by one could be something as below. In this way if the enum receive new values in the feature this code does not need to be changed.
```javascript
if (Object.values(EnrollmentPeriod).includes(enrollmentPeriod)) {
    // Do stuff here
}
```

Reply:
Done.